### DISCUSSION TITLE
Colour for tags like TODO or FIXME

### DISCUSSION THREAD

**Discussion by supertoothy** [zim-desktop-wiki/zim-desktop-wiki#2291] [Q&A] [answered]
For tasks in Zim, I use tags like TODO and FIXME alongside the checkboxes. I find that the tags provide a little more context for tasks.

Is there a way to give these pre-defined labels a different colour by default, so that they stand out? 

I looked under style.conf but could not find the style for task labels.

any pointers?

**Comment by jaap-karssenberg** [comment]
Can you elaborate - e.g. where do you want to see another color ?  Can you
make a mock-up drawing ?

-- Jaap

**Comment by supertoothy** [comment]
This is a screenshot from my zim page.

[url=https://ibb.co/FxGp71Q][img]https://i.ibb.co/5MqZc3Q/zim-todos-230112.png[/img][/url] (To mockup the TODO, I just bold faced it.)

If the TODOs in the begining of each line can be of a slightly different colour, and even font type maybe. This will help me quickly identify the TODO tag as a meta element, and as different from the task description. 

Style.conf allows changing the colour of the tags (@tag) This is great because - instant visibility! The TODOs on the other hand are harder to spot quickly. If we can get a style property for them in style.conf it will be great. (Maybe font properties and colour)

Thank you!

**Comment by jaap-karssenberg** [comment, marked as answer]
This is certainly doable. Would require a plugin that adds a GtkTextTag on
selected key words. This plugin could be generic for any keyword - does not
even need to know about tasklist.

--- Jaap
